lib/repl.js

```diff
@@ -1854,10 +1830,17 @@ function Recoverable(err) {
 ObjectSetPrototypeOf(Recoverable.prototype, SyntaxErrorPrototype);
 ObjectSetPrototypeOf(Recoverable, SyntaxError);
 
+function createREPLServer(prompt, source, eval_, useGlobal, ignoreUndefined, replMode) {
+  return new REPLServer(prompt, source, eval_, useGlobal, ignoreUndefined, replMode);
+}
+
+ObjectSetPrototypeOf(createREPLServer.prototype, REPLServer);
+ObjectSetPrototypeOf(createREPLServer, REPLServer);
+
 module.exports = {
   start,
   writer,
-  REPLServer,
+  REPLServer: createREPLServer,
```

Comment:
This makes the change here semver-major and breaking. 

Before this change:
```js
const repl = require('repl');
const r = repl.REPLServer();
console.log(r instanceof repl.REPLServer); // true
```

After this change:
```js
const repl = require('repl');
const r = repl.REPLServer();
console.log(r instanceof repl.REPLServer); // false
```

Reply:
Oh! You are correct. I can modify it to be non-breaking.